document/CoreFormat.mdk

```diff
@@ -585,38 +589,38 @@ shall either both be present or both be absent.
 
 A `location` object shall contain a property named `analysisTarget` whose value is a `physicalLocation` object (&sect;[#physicalLocation-info])
 that identifies the file that the static analysis tool was instructed to scan.
-This need not be the same as the file where the issue actually occurs. See `issueFile` (&sect;[#location-issueFile]) for more information on this point.
+This need not be the same as the file where the result actually occurs. See `resultFile` (&sect;[#location-resultFile]) for more information on this point.
 
-### `issueFile` property { #location-issueFile }
+### `resultFile` property { #location-resultFile }
 
-A `location` object may contain a property named `issueFile` whose value is a `physicalLocation` object (&sect;[#physicalLocation-info])
-that identifies the file where the static analysis tool detected the issue.
+A `location` object may contain a property named `resultFile` whose value is a `physicalLocation` object (&sect;[#physicalLocation-info])
+that identifies the file where the static analysis tool produced the result.
 
-If the issue actually occurs at the location specified in the `analysisTarget` property,
-then the `issueFile` property shall not be present.
-If the issue occurs in a different file from the one specified in the `analysisTarget` property,
-then the `issueFile` property shall be present.
+If the result actually occurs at the location specified in the `analysisTarget` property,
+then the `resultFile` property shall not be present.
+If the result occurs in a different file from the one specified in the `analysisTarget` property,
+then the `resultFile` property shall be present.
 
 ~ IsoNumberedExample { @IsoNumberedExample=1 }
 Suppose a static analysis tool for C++ source code is instructed to scan the source file _a.cpp_,
 which includes the header file _b.h_,
-and suppose the tool detects an issue in _b.h_.
+and suppose the tool produces a result in _b.h_.
```

Comment:
can you say 'detects a problem in b.h', since this language is appropriate for the example. 'produces a result in b.h' sounds as if we're writing to the file.